glean-core/src/lib.rs

```diff
@@ -402,6 +412,20 @@ impl Glean {
         self.max_events
     }
 
+    /// Gets the next task for an uploader. Which can be either:
+    ///
+    /// Wait - which means the requester should ask again later;
+    /// Upload(PingRequest) - which means there is a ping to upload. This wraps the actual request object;
+    /// Done - which means there are no more pings queued right now.
+    ///
+    /// # Return value
+    ///
+    /// `PingUploadTask` - an enum representing the three possible tasks.
```

Comment:
Same here, if we just say `the possible tasks` we can extend the enum when we need to